docs/resources.md

```diff
@@ -590,18 +590,21 @@ Params that can be added are the following:
 1.  `artifactType`: represent the type of GCS resource. Right now, we support
     following types:
 
-    -   `Archive`:
-        -   Archive indicates that resource fetched is an archive file.
-            Currently, Build GCS resource only supports `.zip` archive.
-        -   It unzips the archive and places all the files in the directory
-            which is set at runtime.
-        -   If `artifactType` is set to `Archive`, `location` should point to a
-            `.zip` file.
+    -   `ZipArchive`:
+	-   ZipArchive indicates that resource fetched is an archive file in
+	    the zip format.
+	-   It unzips the archive and places all the files in the directory
+	    which is set at runtime.
+	-   `Archive` is also supported and is equivalent to `ZipArchive`, but
+	    is deprecated.
+    -   `TarGzArchive`:
+	-   TarGzArchive indicates that resource fetched is a gzipped archive file
+	    in the tar format.
+	-   It unzips the archive and places all the files in the directory
+	    which is set at runtime.
```

Comment:
when i preview this the indentation is a bit off for some reason:

![image](https://user-images.githubusercontent.com/432502/63174826-c0e8c200-c010-11e9-9981-61e425b54d34.png)

Reply:
Huh, weird. I think I had mixed tabs and spaces 😅 

Fixed!